pkg/text/dictionaryitem_test.go

```diff
@@ -0,0 +1,31 @@
+package text_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/fastly/cli/pkg/testutil"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v2/fastly"
+)
+
+func TestPrintDictionaryItem(t *testing.T) {
+	for _, testcase := range []struct {
+		name           string
+		prefix         string
```

Comment:
Should we delete this as we don't actually use it?

Reply:
Deleted